src/pages/Home.vue

```diff
@@ -48,37 +48,14 @@
 
             <div class="tag-list">
               <a
-                href=""
+                v-for="tag in tags"
+                :key="tag"
+                :href="tag"
                 class="tag-pill tag-default"
-              >programming</a>
-              <a
-                href=""
-                class="tag-pill tag-default"
-              >javascript</a>
-              <a
-                href=""
-                class="tag-pill tag-default"
-              >emberjs</a>
-              <a
-                href=""
-                class="tag-pill tag-default"
-              >angularjs</a>
-              <a
-                href=""
-                class="tag-pill tag-default"
-              >react</a>
-              <a
-                href=""
-                class="tag-pill tag-default"
-              >mean</a>
-              <a
-                href=""
-                class="tag-pill tag-default"
-              >node</a>
-              <a
-                href=""
-                class="tag-pill tag-default"
-              >rails</a>
+                @click.prevent="chooseTag(tag)"
```

Comment:
Did You forget to put up this `chooseTag` code? :)

Reply:
Yes, sorry